utox.1

```diff
@@ -39,9 +39,10 @@ status and contacts. It's compatible with other Tox clients.
 \fButox_save\fP is µTox' binary config file, it contains settings such as
 language, proxy options, DPI or logging.
 .SS avatars
-The \fBavatars\fP directory contains other contacts' avatars along with their
-hashes. Avatars are named [public_key].png, avatar hashes are called
-[public_key].hash.
+The \fBavatars\fP directory contains other contacts' avatars. Files are named
+after contacts' public keys (the first 64 characters of the ID).
+.SS [public key].txt
+Those are friends' chat logs. µTox supports only 1v1 chat logging for now.
```

Comment:
Repeating `1 on 1` seems bad to me, is `1v1` invalid in this context? I took it from the clients page on the wiki.